### ISSUE TITLE
Questions about the paper

### ISSUE THREAD

**Issue by lkyee** [XuecaiHu/Meta-SR-Pytorch#1]
Hi,
I have some questions about the paper.Take the RDN-based meta-SR as an example. How is the Feature Learning Module trained? Is the upsampling module removed directly based on RDN x4?In addition, how is the meta-upscale module trained? Is the parameter of the Feature Learning Module fixed and then trained on meta-upscale or a combination of both？
If it is convenient, can you talk about the training pipeline in detail? What is the input and label?
Thank you very much!!(respect to big old)
Look forward to your kind reply!

**Comment by XuecaiHu** [comment]
Thank for your question. I train our Meta-RDN the whole network (FLM and MUM) from scratch.
